[TW-351] Intro to SSO refresh (#4674)
postmanlabs/postman-docs: src/pages/docs/administration/sso/intro-sso.md

> **[SSO is available on Postman Professional and Enterprise plans.](https://www.postman.com/pricing)**

Single sign-on (SSO) services enable you to manage your team’s identity across all the SaaS products that you use.

> SSO services let a user to use one set of credentials (for example, a name or email address and password) to access multiple applications. The service authenticates the user only once for all the applications the user has been given rights to and eliminates further prompts when the user switches applications during the same session.

Reviewer: ```suggestion
With an SSO service, a user can access multiple applications using one set of credentials (for example, an email address and password). The SSO service authenticates the user once for all the applications the user has been given rights to and eliminates further prompts when the user switches applications during the same session.
```

Reviewer: "SSO services let a user to use..." needs rewriting 😄

An example of SSO is Google's sign-in implementation for products like Gmail, YouTube, and Google Drive. Any user who is signed in to one of Google's products is automatically signed in to their other products as well.

## Contents

* [What are the advantages of SSO?](#what-are-the-advantages-of-sso)
* [Prerequisites for SSO with Postman](#prerequisites-for-sso-with-postman)
* [Supported IdPs](#supported-idps)
* [SSO setup for SAML 2.0 compliant IdPs](#sso-setup-for-saml-20-compliant-idps)

## What are the advantages of SSO?

* Removes the need for users to remember and manage multiple passwords.

> * Simplifies users' experience by allowing them to sign in at a single access point and enjoy a seamless experience across multiple applications.

Reviewer: ```suggestion
* Simplifies users' experience by enabling them to sign in at a single access point and work seamlessly across multiple applications.
```

* Increases productivity by significantly reducing the password-related support emails.

> * Reduces phishing and thereby making sure users aren't tricked into giving away sensitive information.

Reviewer: ```suggestion
* Reduces phishing attempts that try to trick users into giving away sensitive information.
```

Enterprise plans:

* [Microsoft AD FS](/docs/administration/sso/microsoft-adfs/)
* [Azure AD](/docs/administration/sso/azure-ad/)

> * [Custom SAML](/docs/administration/sso/custom-saml/)

Reviewer: I assume you'll merge your other PR first so the Custom SAML link here will work?

Author: My plan is to merge them at the same time, and then test on the `develop` branch. What do you think?

Author: Yes, I agree it makes sense to first merge the PR with the Custom SAML page, and then pull the changes into this branch for testing. Thanks!

* [Duo](/docs/administration/sso/duo/)
* [Google Workspace](/docs/administration/sso/google-workspace/)
* [Okta](/docs/administration/sso/okta/)
* [OneLogin](/docs/administration/sso/onelogin/)
* [Ping Identity](/docs/administration/sso/ping-identity/)

## SSO setup for SAML 2.0 compliant IdPs

Most SAML 2.0 compliant identity providers require the same information about the service provider for setup (Postman is the service provider). These values are specific to a Postman team and are available after [configuring SSO](/docs/administration/sso/admin-sso/).

> > While configuring your IdP, make sure the user’s email address is set in attribute statements.

Reviewer 2: Can we highlight this part. This is essential for the SSO set up .

Reviewer 2: The rest of the changes looks good to me .

Author: Thanks for bringing this up. I created ticket `TW-1053` to look into this in more detail.

<!-- -->

> > Postman’s form-based authentication mechanism and Google OAuth 2.0 remain enabled even if SSO is configured for a team.

Reviewer 3: These auth methods are not automatically disabled when SSO is configured but the team admin can disable these if so desired.

Author: This is now updated. Please see my latest commits. Thanks!
